utils/poemgr/Makefile

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: GPL-2.0-only
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=poemgr
+PKG_SOURCE_DATE:=2022-03-19
+PKG_SOURCE_VERSION:=77395242f9b3cb6ef003c5ed74f9b5be78404450
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/blocktrron/poemgr.git
+PKG_MIRROR_HASH:=68034f886af8a4ebfd869af940510f755d660362a6069ac49c1ae6ce16cbd950
+
+PKG_MAINTAINER:=David Bauer <mail@david-bauer.net>
+PKG_LICENSE:=GPL-2.0-only
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/poemgr
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:=+libuci +libjson-c
+  TITLE:=Utility to control PoE ports on the UniFi Flex switch
+endef
+
+define Package/poemgr/conffiles
+/etc/config/usteer
```

Comment:
Nope.

Reply:
This package is already in OpenWrt master branch. See https://github.com/openwrt/packages/pull/18108 So, it needs to be fixed there and here as well.

Reply:
It is already fixed upstream.

Reply:
Yes, but the commit is not backported here, so this makes this PR as stalled.

Reply:
fixed